test/e2e/funcs/feature.go

```diff
@@ -57,6 +57,9 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource/unstructured/composite"
 )
 
+// DefaultPollInterval is the suggested poll interval for wait.For.
+const DefaultPollInterval = time.Millisecond * 500
```

Comment:
not sure if this is going to speed up anything, because the default poll interval in e2e framework is 5s. For sure, it is going to produce more requests against api server.

Reply:
5s is far too much. The apiserver can serve thousands of requests per second. Why waste (developer) time?

Reply:
500ms == 0.5s